Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromCondDB.cc

```diff
@@ -165,13 +165,15 @@ GEMEtaPartition* GEMGeometryBuilderFromCondDB::buildEtaPartition(const RecoIdeal
   float ti = *(shapeStart + 3) / cm;
```

Comment:
If possible, it would be good to remove `#include "CLHEP/Units/GlobalSystemOfUnits.h"` from this file and replace it with `#include "DataFormats/Math/interface/GeantUnits.h"`. Then lines like this one would become:
```
float ti = convertMmToCm(*(shapeStart + 3));
```

Reply:
Yes, ```convertMmToCm``` is applied for the unit converter.